_includes/v22.2/backups/backup-options.md

```diff
@@ -1,7 +1,7 @@
  Option                                                          | Value                   | Description
 -----------------------------------------------------------------+-------------------------+------------------------------
-`revision_history`<a name="with-revision-history"></a>           | N/A                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.
+`revision_history`<a name="with-revision-history"></a>           | [`BOOL`](bool.html) / None                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.<br><br>You can specify a backup with revision history  without any value e.g., `WITH revision_history`. Or, you can explicitly define `WITH revision_history = 'true' / 'false'` — this is useful for setting and unsetting this option when {% if page.name == "alter-backup-schedule.md" %} [applying different options to scheduled backups](#apply-different-options-to-scheduled-backups). {% else %} [altering a backup schedule](alter-backup-schedule.html). {% endif %} 
```

Comment:
Extra space between `history` and `without`, I think?

Maybe specify that ALTER requires this to be explicit, whereas CREATE defaults to true? (Or expand the comment below, perhaps)